docs/API.md

```diff
@@ -477,7 +477,7 @@ minioClient.fGetObject('mybucket', 'photo.jpg', '/tmp/photo.jpg', function(err)
 
 ```
 <a name="putObject"></a>
-### putObject(bucketName, objectName, stream, size, contentType, callback)
+### putObject(bucketName, objectName, stream, size, callback)
```

Comment:
It is better to mention it in docs otherwise no one will know how to use it and it is consistent with our other libs as well.